sql/lens_metadata.sql

```diff
@@ -0,0 +1,19 @@
+SELECT
+  scholarly.lens_id AS id,
+  MAX(title) as title,
+  MAX(abstract) as abstract,
+  MAX(LOWER(id.value)) AS clean_doi,
+  MAX(year_published) as year,
+  ARRAY_AGG(author.last_name) AS last_names,
+  ARRAY_AGG(reference.lens_id) AS references
```

Comment:
Again please sanity check me - they're unusual in that they provide both references and citations

Reply:
Did this execute successfully for you? I get `Cannot query rows larger than 100MB limit.` which led me [here](https://stackoverflow.com/questions/70906402/cannot-query-rows-larger-than-100mb-limit) suggesting the issue is with the size of the values over which we're applying `array_agg`?

Reply:
Ugh I must have somehow forgotten to run this one. Sorry about that - can you take a look at the updated version I've added in the latest commit?